v1/tasks/errors.go

```diff
@@ -5,29 +5,8 @@ import (
 	"time"
 )
 
-// ErrRetryTaskLater ...
-type ErrRetryTaskLater struct {
```

Comment:
We no longer use `ErrRetryTaskLater` -- we always use `ErrKeepAndRetryTaskLater`